forge.lic

```diff
@@ -45,8 +45,15 @@ class Forge
     @bag = settings.crafting_container
     @belt = settings.forging_belt
     @hammer = settings.forging_tools.find { |item| /hammer/ =~ item }
+    @tong_shovel = settings.use_adjustable_tong
 
     @item = args.noun
+    
+    if @tong_shovel
+      @tool = 'tong'
```

Comment:
I see quite a few other places that 'tongs' are used. Should they also reference `@tool`?